src/runtime/pkg/katautils/config.go

```diff
@@ -329,7 +329,7 @@ func (h hypervisor) defaultVCPUs() uint32 {
 
 func (h hypervisor) defaultMaxVCPUs() uint32 {
 	numcpus := uint32(goruntime.NumCPU())
-	maxvcpus := vc.MaxQemuVCPUs()
+	maxvcpus := vc.MaxVCPUs()
```

Comment:
@sameo, I'm adding the comment here, but it's rather related to 9d8fe3e0e454e6e17a27d5789ee1d69c2f615649 than to this specific part.

I like you're making things that are QEMU specific less QEMU specific, which is a good thinng.  But I wonder whether the values should be coming from GoVMM (that currently lives inside our repo), and I think this refactor is a nice opportunity to also touch this.

WDYT?

Reply:
I agree. It should look better now, hopefully.
I added those functions to the govmm package and they're all gone from virtcontainers.

Let me know if that was not what you were looking for.